gcp-bundle/LICENSE

```diff
@@ -201,361 +201,484 @@
+License (from POM): The Apache License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0.txt
+
+--------------------------------------------------------------------------------
+
+Group: io.opentelemetry.semconv  Name: opentelemetry-semconv  Version: 1.27.0-alpha
+Project URL (from POM): https://github.com/open-telemetry/semantic-conventions-java
+License (from POM): The Apache License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0.txt
+
+--------------------------------------------------------------------------------
+
+Group: io.perfmark  Name: perfmark-api  Version: 0.27.0
+Project URL (from POM): https://github.com/perfmark/perfmark
+License (from POM): Apache 2.0 - https://opensource.org/licenses/Apache-2.0
 
 --------------------------------------------------------------------------------
 
 Group: javax.annotation  Name: javax.annotation-api  Version: 1.3.2
-Project URL: https://javaee.github.io/glassfish
-Project URL: http://jcp.org/en/jsr/detail?id=250
-License: CDDL + GPLv2 with classpath exception - https://github.com/javaee/javax.annotation/blob/master/LICENSE
+Project URL (from manifest): https://javaee.github.io/glassfish
+Project URL (from POM): http://jcp.org/en/jsr/detail?id=250
+License (from POM): CDDL + GPLv2 with classpath exception - https://github.com/javaee/javax.annotation/blob/master/LICENSE
 
 --------------------------------------------------------------------------------
 
-Group: org.checkerframework  Name: checker-qual  Version: 3.33.0
-Project URL: https://checkerframework.org/
-License: The MIT License - http://opensource.org/licenses/MIT
+Group: org.checkerframework  Name: checker-qual  Version: 3.48.3
+Manifest License: MIT (Not packaged)
+Project URL (from POM): https://checkerframework.org/
+License (from POM): The MIT License - http://opensource.org/licenses/MIT
 
 --------------------------------------------------------------------------------
 
-Group: org.codehaus.mojo  Name: animal-sniffer-annotations  Version: 1.23
-License: MIT license - https://spdx.org/licenses/MIT.txt
-License: The Apache Software License, Version 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
+Group: org.codehaus.mojo  Name: animal-sniffer-annotations  Version: 1.24
+License (from POM): MIT license - https://spdx.org/licenses/MIT.txt
+License (from POM): The Apache Software License, Version 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
 Group: org.conscrypt  Name: conscrypt-openjdk-uber  Version: 2.5.2
-Project URL: https://conscrypt.org/
-License: Apache 2 - https://www.apache.org/licenses/LICENSE-2.0
+Project URL (from POM): https://conscrypt.org/
+License (from POM): Apache 2 - https://www.apache.org/licenses/LICENSE-2.0
+
+--------------------------------------------------------------------------------
+
+Group: org.slf4j  Name: slf4j-api  Version: 2.0.16
```

Comment:
Same here, imho, it's not correct because slf4j-api is excluded (https://github.com/apache/iceberg/blob/main/gcp-bundle/build.gradle#L42) and it's not part of the distributed jar file.

See this PR: https://github.com/apache/iceberg/pull/12052